python/cuml/dask/datasets/regression.py

```diff
@@ -134,9 +134,9 @@ def _shuffle(client, rs, X, y, chunksizes, n_features,
     return da.concatenate(X_dela, axis=0), da.concatenate(y_dela, axis=0)
 
 
-def _convert_C_to_F_order(client, X, chunksizes, n_features, dtype):
+def _convert_order(client, X, chunksizes, order, n_features, dtype):
```

Comment:
Maybe `output_order` instead of order? I wasn't really sure what to expect from this argument.